src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AdditionalServices.feature

```diff
@@ -183,9 +183,23 @@ Scenario: Additional Services - Go back post save
 	When the User chooses to go back
 	Then the Additional Service dashboard is presented
 
-Scenario: Additional Services - Values populated after editing and saving
+Scenario: Additional Services - Values populated after editing and saving - Flat List Price Variable (Patient Numbers)
 	Given the edit Additional Service form for flat list price with variable (patient numbers) order type is presented
 	And the Order Form for the existing order is presented
 	And the User is able to manage the Additional Services section
 	When the User chooses to edit the saved Additional service	
+	Then the pricing values will be populated with the values that was saved by the User
+
+Scenario: Additional Services - Values populated after editing and saving - Flat List Price Declarative
+	Given the edit Additional Service form for flat list price with declarative order type is presented
+	And the Order Form for the existing order is presented
```

Comment:
you shouldn't need this step if you're using the one above should you? as you're already on the edit price form page